src/jdk.jartool/share/classes/sun/tools/jar/Main.java

```diff
@@ -1524,12 +1506,12 @@ void list(InputStream in, String files[]) throws IOException {
      * Lists contents of JAR file, via ZipFile.
      */
     void list(String fname, String files[]) throws IOException {
```

Comment:
```suggestion
    void list(String fname, String[] files) throws IOException {
```

Synchronize with the array syntax change in (new) line 1490.

Reply:
You are right - I missed this line and a few other similar declarations. I've now updated the PR to address them.